drivers/i2c/Kconfig

```diff
@@ -218,7 +220,7 @@ config I2C_1
 config I2C_1_NAME
 	string "Port 1 device name"
 	default "I2C_1"
-	depends on I2C_1
+	depends on I2C_1 && !HAS_DTS_I2C
 
 config I2C_1_DEFAULT_CFG
```

Comment:
I wanted to get the base I2C stuff in for DT, I have idea's on how we can we than use DT to replace I2C_n_DEFAULT_CFG for HAS_DTS_I2C enabled platforms